src/Models/Laravel/Attribute.php

```diff
@@ -35,6 +38,35 @@ class Attribute extends BaseModel
      */
     protected $guarded = [];
 
+    /**
+     * The attribute key as String.
+     */
+    public function keyString(): AttributeCasts
+    {
+        return new AttributeCasts(
+            get: fn () => Hex::safeConvertToString($this->key)
+        );
+    }
+
+    /**
+     * The attribute value as String.
+     */
+    public function valueString(): AttributeCasts
+    {
+        return new AttributeCasts(
+            get: function () {
+                $key = Hex::safeConvertToString($this->key);
+                $value = Hex::safeConvertToString($this->value);
```

Comment:
If we are able to store binary/non-string data from the chain as hex, do we have ways to handle detecting that and returning something meaningful in the API?  Would be good to test that too.

Reply:
Created a ticket